takeshape/penny
# delivery status components

### Screenshots

<img width="645" alt="Screenshot 2022-09-15 at 13 13 16" src="https://user-images.githubusercontent.com/277401/190498595-499a30fc-446b-4474-8ab6-c9a39b4b49c7.png">

### Test Plan (Steps to test):

1. Create a subscription, or use an existing
1. Log in to the Shopify admin for DSP. Find one of your subscription orders and manually fulfill it. Give a shipping company name and tracking number.
1. Check on that order in your Penny Accounts > Subscriptions section. You should see the order at the fulfilled status, and be able to click a `Tracking Info` button on the order entry.
